common/assets/users-groups-roles.tmpl

```diff
@@ -0,0 +1,156 @@
+{
+  "objects": [
+  {{ if eq .objectType "user" }}
```

Comment:
Maybe not a big deal for now but it looks like this template would produce a lot of needless newlines in its output.
I would expect this line, for example, to be:
```
  {{- if eq .objectType "user" }}
```

Also, I find the pipeline syntax in template function to produce more readable code. For example:
```
  {{- if .objectType | eq "user" }}
```

But that might be a matter of preference.